pages/configuration/elasticsearch.rst

```diff
@@ -8,108 +8,77 @@ We strongly recommend to use a dedicated Elasticsearch cluster for your Graylog
 
 If you are using a shared Elasticsearch setup, a problem with indices unrelated to Graylog might turn the cluster status to YELLOW or RED and impact the availability and performance of your Graylog setup.
 
-.. important:: Graylog currently does not work with Elasticsearch clusters using the License or `Shield <https://www.elastic.co/guide/en/shield/2.3/index.html>`__ plugin.
-
 
 Elasticsearch versions
 ======================
 
-Graylog hosts an embedded Elasticsearch node which is joining the Elasticsearch cluster as a client node.
-
-The following table provides an overview over the Elasticsearch version in Graylog:
-
-===============  =====================
-Graylog version  Elasticsearch version
-===============  =====================
-1.2.0-1.2.1      1.7.1
-1.3.0-1.3.3      1.7.3
-1.3.4            1.7.5
-2.0.0            2.3.1
-2.0.1-2.0.3      2.3.2
-2.1.0-2.1.3      2.3.5
-2.2.0-2.2.3      2.4.4
-===============  =====================
-
-.. caution:: Graylog 2.x **does not** work with Elasticsearch 5.x!
+Since Graylog uses the HTTP protocol to connect to your Elasticsearch cluster, it does not have a hard requirement for the Elasticsearch version anymore. We can safely assume that any version starting from 2.x is working.
 
+.. caution:: Graylog 2.3 **does not** work with Elasticsearch 6.x yet!
 
 Configuration
 =============
 
+.. caution:: As Graylog has recently switched from an embedded Elasticsearch node client to a lightweight HTTP client, please check the `upgrade notes <https://github.com/Graylog2/graylog2-server/blob/bb820e8/UPGRADING.rst#graylog-switches-to-elasticsearch-http-client>`_ how to migrate your configuration if you are switching from an earlier version.
```

Comment:
As Graylog has switched from an embedded Elasticsearch node client to a HTTP client with Version 2.3, ...